### PR TITLE
Get member emails

### PR DESCRIPTION
**Pull request by Jezerig**
The user can choose which seperator they use between emails

### CODE REVIEW

**Review by Enlarem — COMMENTED** [review]
image

Maybe something like this for the title would be better ? Otherwise the functionality works really well.

**Comment by Nesssu** [comment]
Could also be a good functionality to copy the email by clicking it on the list.

**Comment by jannejjj** [comment]
> Could also be a good functionality to copy the email by clicking it on the list.

I think so too, but I thinkl we can do that in a separate ticket, it'd be nice to get this one out of the door first :D

File: client/src/components/Members.js [hunk 1]
```diff
@@ -13,13 +16,51 @@ function Members(props) {
   const [displayMembers, setDisplayMembers] = useState([{}]);
   const [loading, setLoading] = useState(true);
   const [update, setUpdate] = useState(false);
+  const [getEmailsModal, setGetEmailsModal] = useState(false);
+  const [seperator, setSeperator] = useState("");
```
Comment (jannejjj) [inline client/src/components/Members.js:20]: Don't want to nitpick, but i think it shoulöd be "separator" :d

**Comment by jannejjj** [comment]
Looks great!

**Review by jannejjj — APPROVED** [review]

**Pull request merged**